mailet/standard/src/main/java/org/apache/james/transport/mailets/MimeDecodingMailet.java

```diff
@@ -84,8 +84,8 @@ public void service(Mail mail) throws MessagingException {
         Serializable attributeContent = mail.getAttribute(attribute);
         if (! (attributeContent instanceof Map)) {
             LOGGER.debug("Invalid attribute found into attribute "
-                    + attribute + "class Map expected but "
-                    + attributeContent.getClass() + " found.");
+                    + "{} class Map expected but "
+                    + "{} found.", attribute, attributeContent.getClass());
             return ImmutableMap.of();
```

Comment:
Maybe you could inline the string now there is no concatenation anymore

Reply:
Done